CHANGELOG.md

```diff
@@ -20,6 +20,15 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - Inverse/Selection color is now modelled after XTerm/VTE instead of URxvt to improve consistency
 - First click on unfocused Alacritty windows is no longer ignored on platforms other than macOS
+- Standalone terminfo definition:
+    - The alacritty terminfo definitions no longer inherit from the systems
+      `xterm` definitions.
+    - The terminfo entry `alacritty` replaces the old `alacritty` and
+      `alacritty-256color` (both were actually 256 colors).
+    - There is a new entry called `alacritty-direct` which advertises 24-bit
+      color support
+    - The default `TERM` value is no longer static; the `alacritty` entry is
+      used if available, otherwise the `xterm-256color` entry is used instead.
```

Comment:
There's a few minor changes I would propose with this changelog entry.

First of all the section about `alacritty-direct` should probably be moved to the added features. There also shouldn't be any punctuation at the end of lines. 

I'd also change the first point about not inheriting from the system's `xterm` definitions to something that shows better what the actual change is for the user (feel free to reword if you have a better idea):
`The Alacritty terminfo definition no longer requires the 'xterm' definition to be present on the system`

I'd also move the `alacritty-256color` mention to the `Removed` section. You can still mention that it is now replaced by the `alacritty` terminfo file, since that is also 256 color, but the actual effect on the user is mostly that the `alacritty-256color` terminfo definition has been removed, so if they were using it they need to change it.

Since that only leaves two points in the `Changed` section of the changelog, I'd also recommend getting rid of the `Standalone terminfo definition:` headline.

Reply:
Done